requirements.txt

```diff
@@ -1,8 +1,5 @@
 alphashape
-numpy~=1.19
 pandas
-cython
-cpython

```

Comment:
you moved numpy and cython in to pre-requirements.txt
but not cpython.
any specific reason for not having that in pre-requirements.txt?

Reply:
I believe CPython was incorrectly included in requirements to begin with. CPython is an implementation of the python language, whereas Cython is a superset to the Python language. I believe somewhere someone has accidentally included CPython as a requirement instead of Cython by mistake. It is not depended on by Onko or any of its requirements. The CPython package on pypi appears to have nothing to do with "real" CPython.